pytest.ini

```diff
@@ -1,4 +1,4 @@
 #Configuration file for py.test
 [pytest]
-python_files=Test_*.py
+python_files=Test_*.py assert*.py
```

Comment:
What's the case?

Reply:
The assertingUtils contains an assert statement. If it is not in this list pytest will not inject itself in the statement and the output will be rather useless if the asserts are failing in this file.